contents/blog/why-we-made-surveys.md

```diff
@@ -0,0 +1,41 @@
+---
+
+Today, we’ve announced user surveys are out of beta and in general release, complete with [new pricing](/pricing). Short version? Pricing is usage-based, with a free tier for small projects and an improved offer for everyone who participated in the beta. 
+
+Launches like this are always exciting for us, but this one is _especially_ notable because it firmly establishes PostHog as an all-in-one tool for building better products. 
+
+Why is this important? Because spreading everything out across multiple platforms _sucks_. Replays in HotJar, feature flags in LaunchDarkly, analytics in Amplitude (but not your SQL insights, those go somewhere else), and experiments in Google Optimize? That only lays a foundation for frustration, incompatibility, and inconsistency.
+
+It’s also a pain having to log in and out of new platforms all the time. 
+
+Imagine instead if you could create surveys triggered by the same events you track in your analytics, or if you could trace a survey response back to a session replay in a single platform. Imagine if you could test changes with an A/B experiment and launch surveys alongside them to collect feedback, then turn that into a feature flag and roll it out without ever switching tools.
+
+Imagine if you had your qualitative and quantitative data side-by-side, along with everything you need to dissect it and act on your findings. 
+
+That’s the vision we have for PostHog — an all-in-one platform where you can research users, observe usage, analyze data, test ideas, deploy changes, and measure impact consistently, and easily. No more tool juggling, compatibility wrestling, unclear pricing, or botched integrations; just a single, open-source platform that does it all.
+
+Surveys aren’t [our defining feature](/handbook/company/values#we-havent-built-our-defining-feature-yet), but they’re an essential part of this vision because they bring qualitative data into PostHog alongside the quantitative and intuitive data we already collected through events and session replays. 
+
+They're _so_ essential that we've heavily prioritized them. Here's a look at the progress since the beta started, and the pace of when new features dropped:
+
+- **[June 20th:](/changelog/2023#surveys-beta-released)** Beta launched with open text surveys
+- **[July 25th:](/changelog/2023#emoji-and-number-surveys-types-released)** Emoji and number rating survey types added
+- **[August 8th:](/changelog/2023#multiple-choice-surveys-released)** Multiple choice surveys added
+- **[August 18th:](/changelog/2023#wait-periods-now-available-for-surveys)** Wait periods and thank you messages added
+- **[September 14th:](/changelog/2023#automatic-nps-scoring-added-to-surveys)** Automatic NPS calculation added
+
+Basically, we’ve made sure that surveys are perfect for the most common use-cases, including [NPS surveys](/tutorials/nps-survey), customer satisfaction surveys, and [product-market fit surveys](/blog/measure-product-market-fit). 
```

Comment:
```suggestion
## What can you do with surveys in PostHog?

Since we shipped the [initial beta in June](/changelog/2023#surveys-beta-released) with open text surveys, we've added:

- **[July 25:](/changelog/2023#emoji-and-number-surveys-types-released)** Emoji and number rating survey types
- **[August 8:](/changelog/2023#multiple-choice-surveys-released)** Multiple choice surveys
- **[August 18:](/changelog/2023#wait-periods-now-available-for-surveys)** Wait periods and thank you messages
- **[September 14:](/changelog/2023#automatic-nps-scoring-added-to-surveys)** Automatic NPS calculation

These features make surveys perfect for the most common use-cases, including [NPS surveys](/tutorials/nps-survey), customer satisfaction surveys, and [product-market fit surveys](/blog/measure-product-market-fit). 
```

Reply:
I don't think this suggested header really adds much, so I've tweaked it.